src/caret_analyze/value_objects/timer.py

```diff
@@ -67,28 +131,79 @@ def __init__(
         callback_info: TimerCallbackStructValue | None,
         construction_order: int
     ) -> None:
+        """
+        Construct an instance.
+
+        Parameters
+        ----------
+        node_name : str
+            Node name.
+        period_ns : int
+            Period of the timer.
+        callback_info : TimerCallbackStructValue | None
+            Static info of callback.
+        construction_order : int
+            Order of instance creation within the identical node.
+
+        """
         self._node_name: str = node_name
         self._period_ns: int = period_ns
         self._callback_value = callback_info
         self._construction_order = construction_order
 
     @property
     def node_name(self) -> str:
+        """
+        Get node name.
+
+        Returns
+        -------
+        str
+            Node name.
+
+        """
         return self._node_name
 
     @property
     def period_ns(self) -> int:
+        """
+        Get period.
+
+        Returns
+        -------
+        int
+            Period of the timer.
+
+        """
         return self._period_ns
 
     @property
     def callback_name(self) -> str | None:
+        """
+        Get callback name.
+
+        Returns
+        -------
+        str | None
+            Callback name
```

Comment:
```suggestion
            Callback name.
```

Reply:
Fixed it in [21d4698](https://github.com/rokamu623/CARET_analyze/pull/25/commits/21d46989ce2b8b96412934225b795d82987f7a89).